static/js/dashboard.js

```diff
@@ -24,6 +21,10 @@ if (!Object.entries) {
 // requirement for dialog-polyfill
 import 'style!css!dialog-polyfill/dialog-polyfill.css';
 
+// material-ui requirement
+let injectTapEventPlugin = require("react-tap-event-plugin");
+injectTapEventPlugin();
```

Comment:
is this required just for the material-ui component to work? I'm just wondering if we're worrying about the usability of MM on a touch device at all.